style/javascript/sample.js

```diff
@@ -0,0 +1,7 @@
+object = {spacing: false}
```

Comment:
I was actually referring to single-line spacing. 

I prefer this:

```js
{ spacing: true }
```

to this:

```js
{spacing: false}
```